torchtune/utils/collate.py

```diff
@@ -71,3 +71,57 @@ def padded_collate(
             value=padding_idx,
         )
     return input_ids, labels
+
+
+def padded_collate_dpo(
+    batch: List[TokenPair],
```

Comment:
hm, why haven't we been using this type across the codebase? @joecummings any thoughts on keeping this abstraction or getting rid of it?

Reply:
ok with leaving it in this file only or dropping it completely